conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/ConjureUndertowRuntime.java

```diff
@@ -132,8 +141,51 @@ public Builder requestArgHandler(RequestArgHandler value) {
             return this;
         }
 
+        @CanIgnoreReturnValue
+        public Builder onSetRequestToken(BiConsumer<HttpServerExchange, Optional<UnverifiedJsonWebToken>> callback) {
```

Comment:
Let's model this after the `public Builder requestArgHandler(RequestArgHandler value) {` method above and define a custom interface. Perhaps something like:
```java
public Builder jsonWebTokenHandler(JsonWebTokenHandler value) {
```

A custom `JsonWebTokenHandler` interface should be a bit easier for folks to understand at a glance, and we can javadoc it with relevant info.

Reply:
fixed